setup/README.md

```diff
@@ -0,0 +1,20 @@
+# Installation script
+
+This folder contains a script that is useful for the installation of the k3s cluster locally with just one command.
+
+The script will perform the following tasks:
+1. Create a management user (mgmt)
+2. Install Ansible
+   1. Check if the Ansible is 2.10, if so, install the latest version
+3. Clone Ansible GIT repository
```

Comment:
I would simply "download" (e.g., wget) the repository with the Ansible scripts required to install the box, instead of _cloning_. This has the advantage that we don't need to install git on the mini-pc, which is not strictly required.